deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,75 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  origin {
+    domain_name = "${module.origin.site_bucket}.s3.amazonaws.com"
+    origin_id   = "originEastId"
+  }
+
+  enabled             = true
+  http_version        = "http2"
+  comment             = "EchoLocator (${var.environment})"
```

Comment:
Potentially useless nit, but is it `EchoLocator`? I don't know, but wanted to check.

Reply:
Potentially unrelated, but I just realized that this should probably be `"${var.project} (${var.environment})"`.

Reply:
We've been using `ECHOLocator` or `ECHO Locator`.

Reply:
Ooh nice, thanks @ddohler!

Reply:
Yeah, comment was purely about the spacing and casing. Going with one of the ones Derek suggests sounds good.

Reply:
Got it, should be updated now.